### PR TITLE
Build requires Qt5DBus

### PR DESCRIPTION
**Pull request by monich**
I also suggest to drop the .yaml file. AFAICT it's useless.

### CODE REVIEW

File: harbour-fernschreiber.pro [hunk 1]
```diff
@@ -104,3 +104,5 @@ HEADERS += \
     src/tdlibsecrets.h \
     src/tdlibwrapper.h
 
+OTHER_FILES += \
+    rpm/$${TARGET}.spec
```
Comment (Wunderfitz) [inline harbour-fernschreiber.pro:108]: Any particular reason why to add OTHER_FILES and only to add the spec file here?

**Review by Wunderfitz — COMMENTED** [review]
Thanks for adding the DBus build dependency. However, please clarify why OTHER_FILES and only the spec is needed...

File: harbour-fernschreiber.pro (hunk 1 above)
Comment (monich) [reply harbour-fernschreiber.pro:108]: If I started adding other files, then I would have to add all other files too, including the .yaml thing which I think is useless. Adding two out of three files would look strange, and what if you want to keep the .yaml file? So I just added the file which I edited ))

Comment (Wunderfitz) [reply harbour-fernschreiber.pro:108]: But why is the OTHER_FILES section needed at all? All elements are listed in DISTFILES, so what's the point of OTHER_FILES?

Comment (monich) [reply harbour-fernschreiber.pro:108]: Just for convenience of editing, I don't think it has any other meaning

Comment (Wunderfitz) [reply harbour-fernschreiber.pro:108]: :D Well, in this case I might consider deleting this one again. Let's see!

Comment (monich) [reply harbour-fernschreiber.pro:108]: Well, it's up to you. Spec isn't being edited that often. I personally prefer to have all files included into the project and accessible directly from QtCreator but it's a matter if taste and habits of course.

**Pull request merged**